### PR TITLE
Adding code of conduct

### PR DESCRIPTION
**Pull request by bigethan**
This repo has been nothing but happy, so this isn't an attempt to try and fix something that's gone awry.  Just being progressive in thinking about OSS communities and this one that I really like :-)

### CODE REVIEW

**Comment by alexindigo** [comment]
Is it reaction to the troll in underscore+lodash thread? :)

**Comment by bigethan** [comment]
A reaction to inconsiderate behavior over years leading us to a place where mostly only white men are in tech. Change has to come from somewhere sometime.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. Project maintainers who do not follow the Code of Conduct may be removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by opening an issue or contacting one or more of the project maintainers.
+
+This Code of Conduct is adapted from the [Contributor Covenant](http:contributor-covenant.org), version 1.1.0, available at [http://contributor-covenant.org/version/1/1/0/](http://contributor-covenant.org/version/1/1/0/)
```
Comment (alexindigo) [inline CODE_OF_CONDUCT.md]: Missing couple of slashes in `http:contributor-covenant.org`

Comment (bigethan) [inline CODE_OF_CONDUCT.md]: heh:  https://github.com/CoralineAda/contributor_covenant/commit/c1f51a666afabe92e86cc358abb0d4300483a724

**Comment by saponifi3d** [comment]
seems legit. i'm down with keeping this a happy community :)

**Comment by bigethan** [comment]
Thanks :-)

**Pull request merged**